docs/build/eps.md

```diff
@@ -624,7 +640,7 @@ Dockerfile instructions are available [here](https://github.com/microsoft/onnxru
 
 #### Build Phython Wheel
 
-`./build.sh --config Release --build --build_wheel --parallel --use_migraphx --migraphx_home /opt/rocm`
+`./build.sh --config Release --build_wheel --parallel --use_migraphx --migraphx_home /opt/rocm`
```

Comment:
Can you update your branch to latest gh-pages so that only relevant openvino changes are reflecting for review